### PR TITLE
[terra-clinical-data-grid][terra-clinical-item-collection] Updated keycode-js to v2

### LINKED ISSUE #656: [terra-clinical-data-grid] Update KeyCodes dependency
# Feature Request

## Description
keycodes dependency is out-of-date and can be updated to v2
Looks like terra-clinical-data-grid was missed on the original issue logged to update to v2: https://github.com/cerner/terra-core/issues/2606

PR to reference: https://github.com/cerner/terra-core/pull/2616

## Additional Context / Screenshots

```
WARNING in keycode-js
  Multiple versions of keycode-js found:
    1.0.4 ./~/terra-clinical-data-grid/~/keycode-js
    2.0.3 ./~/keycode-js
```

### PR DESCRIPTION
**Pull request by gabeparra01**
### Summary
The open issue #656 identified an older version (v1) of keycode-js still being used in terra-clinical-data-grid. I saw the same older version being used in terra-clinical-item-collection so I updated that to version 2 as well.

Closes #656

### Deployment Link

https://terra-clinical-deployed-pr-#.herokuapp.com/

### Testing
Ran Jest tests and WDIO tests locally for terra-clinical-data-grid and terra-clinical-item-collection. None of the snapshots or screenshots needed to be updated.

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-clinical-data-grid/src/DataGrid.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ResizeObserver from 'resize-observer-polyfill';
 import ContentContainer from 'terra-content-container';
 import { injectIntl, intlShape, FormattedMessage } from 'react-intl';
 
-import KeyCode from 'keycode-js';
+import * as KeyCode from 'keycode-js';
```
Comment (yuderekyu) [inline packages/terra-clinical-data-grid/src/DataGrid.jsx:11]: i know keycode-js is relatively small. but it's probably best to only import what we use, reducing import size
```suggestion
import { KEY_SHIFT, KEY_TAB, KEY_TAB } from 'keycode-js';
```

Comment (tbiethman) [reply packages/terra-clinical-data-grid/src/DataGrid.jsx:11]: The imported code would only be minimized if it is built with tree shaking enabled in the webpack config, but we should still change this to better support that configuration.

Comment (gabeparra01) [reply packages/terra-clinical-data-grid/src/DataGrid.jsx]: Sure sounds good. Removed wildcard imports for keycode-js: https://github.com/cerner/terra-clinical/pull/730/commits/7134b1ed4fa10b287f58607f9e1a251c4f6edd60

File: packages/terra-clinical-data-grid/package.json [hunk 2]
```diff
@@ -29,7 +29,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "keycode-js": "^1.0.4",
+    "keycode-js": "^2.0.1",
```
Comment (mjhenkes) [inline packages/terra-clinical-data-grid/package.json:32]: keycode-js 3.x is out. Should we go straight to that?

Comment (mjhenkes) [reply packages/terra-clinical-data-grid/package.json:32]: https://www.npmjs.com/package/keycode-js

Comment (gabeparra01) [reply packages/terra-clinical-data-grid/package.json]: Updated to 3.x:
https://github.com/cerner/terra-clinical/pull/730/commits/2aac14f19512ab431f1c804266202937f89a6788
https://github.com/cerner/terra-clinical/pull/730/commits/33994fb66a5135eec549530a32f367cf9bfcf32b

**Review by mjhenkes — APPROVED** [review]

**Comment by mjhenkes** [comment]
@gabeparra01 thanks for the contribution!

File: packages/terra-clinical-data-grid/src/DataGrid.jsx (hunk 1 above)
Comment (yuderekyu) [reply packages/terra-clinical-data-grid/src/DataGrid.jsx]: that's good to know!

**Review by yuderekyu — APPROVED** [review]

**Pull request merged**